servicetest/dbus/test_dbus.py

```diff
@@ -98,7 +98,6 @@ def test_query_in(self):
             finally:
                 ca.terminate()
 
-    @pytest.mark.skip(reason="See reported issue about this")
```

Comment:
Ok, removed comment, but what report? Would be nice with a reference in the future.